server/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "lint": "eslint .",
     "build": "rimraf dist/ && babel ./ --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
-    "start": "npm run build && nodemon dist/index.js"
+    "start": "npm run build && node dist/index.js"
```

Comment:
Using `nodemon` here would help in the development process, as it watches for file changes and reloads the server automatically. Could you explain why you removed it?

Reply:
In this case, we need to make a new build every time we add something in the server project. Babel transpile here ES6 code to the older version. I just didn't found an easier way to do this and I will gladly change it :)